main/frame.js

```diff
@@ -0,0 +1,63 @@
+class Frame {
+  constructor() {
+    this.score = 0;
+  }
+
+  addRoll(points) {
```

Comment:
I see that you have functions to add roll and add bonus on both classes, maybe there is a way to DRY this a bit?